frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -267,11 +330,110 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     );
   };
 
+  const isOpenshiftNamespace = (ns: string): boolean => {
+    return _.startsWith(ns, 'openshift-')
+  };
+
+  const createNamespaceAlert = () => {
+    const monitoring = <div style={{ marginLeft: '20px' }}>
+      <Checkbox
+        id={selectedTargetNamespace}
+        label="Enable operator recommended cluster monitoring on this namespace"
+        onChange={(val) => {
+          setEnableMonitoring(val)
+        }}
+        isChecked={enableMonitoring}
+        name="enableNamespaceMonitoring"
+      />
+      <small>Note: Enabling monitoring will allow any operator or workload running on this namespace to contribute metrics to the cluster metric set.</small>
+    </div>
+
+    if (!targetNamespaceExists) {
+      return <>
+        <div style={{ marginTop: '20px' }}>
+          <Alert
+            isInline
+            className="co-alert co-alert--scrollable"
+            variant="info"
+            title="Namespace creation"
+          >
+            <div className="co-pre-line">Namespace <b>{forcedTargetNamespace}</b> does not exist and will be created.</div>
+          </Alert>
+        </div>
+        {isOpenshiftNamespace(selectedTargetNamespace) && monitoring}
+      </>
+    }
+  };
+
+  const globalNamespaceInstallMode = () => {
+    return <>
+      <Dropdown
+        id="dropdown-selectbox"
+        dropDownClassName="dropdown--full-width"
+        menuClassName="dropdown-menu--text-wrap"
+        items={items}
+        title={<ResourceName kind="Project" name={selectedTargetNamespace === forcedTargetNamespace ? `${selectedTargetNamespace} (Operator recommended)` : selectedTargetNamespace} />}
+        disabled={Object.keys(items).length === 1 ? true : false}
+        selectedKey={selectedTargetNamespace}
+        onChange={(ns: string) => {
+          setTargetNamespace(ns);
+          setCannotResolve(false);
+        }}
+      />
+      {createNamespaceAlert()}
+    </>
+  };
+
+  const singleNamespaceInstallMode = () => {
+    if (forcedTargetNamespace) {
+      return <div style={{ marginBottom: '20px' }}>
+      <RadioInput
+        onChange={(e) => {
+          setUseRecommendedNamespace(true)
+          setTargetNamespace(forcedTargetNamespace);
+        }}
+        value={forcedTargetNamespace}
+        checked={useRecommendedNamespace}
+        title="Operator recommended namespace:"
```

Comment:
Does PF let you pass a React node as a title?

Reply:
it does. Screen:
<img width="350" alt="Screen Shot 2020-01-06 at 16 17 29" src="https://user-images.githubusercontent.com/1668218/71827247-1dbc4600-30a0-11ea-8163-14dd50fe1740.png">